src/xdocs/config.xml

```diff
@@ -26,21 +26,23 @@
 
       <ul>
 
-        <li><em>FileSetChecks</em> - modules that take a set of input
+        <li><em>File Set Checks</em> - modules that take a set of input
         files and fire violation messages.</li>
 
         <li><em>Filters</em>
 
         - modules that filter audit events,
         including messages, for acceptance.</li>
 
-        <li><em>AuditListeners</em> - modules that report accepted events.</li>
+        <li><em>Audit Listeners</em> - modules that report accepted events.</li>
 
       </ul>
 
       <p>
         Many checks are submodules of the <em>TreeWalker</em>
-        FileSetCheck module. The TreeWalker operates by separately
+        FileSetCheck module. FileSetCheck TreeWalker checks individual
+        Java source files and defines properties that are applicable
+        to checking such files. The TreeWalker operates by separately
```

Comment:
FileSetCheck --> File Set Check

And I do not understand new sentence. Better to remove it. All details of what is File Set Checks described in special section

Reply:
Alright!

Reply:
@romani what if we include File set Check as 'File set Check'. within commas.

Reply:
Please look at it now.

Reply:
Although I've not committed the changes which I suggested above since I wanted to take your suggestion. 

What I want to say was that: Instead of including the name of module as` File Set Check`, it'd be slightly better if we can include the name of the module in the documentation as `'File Set Check`.

WDYT?

Reply:
>including the name of module as File Set Check

There is no such module. It is the for group of modules of specific type